clang/lib/3C/ProgramInfo.cpp

```diff
@@ -139,10 +139,10 @@ static void getVarsFromConstraint(ConstraintVariable *V, CAtoms &R) {
     if (FVConstraint *FVC = PVC->getFV())
       getVarsFromConstraint(FVC, R);
   } else if (auto *FVC = dyn_cast<FVConstraint>(V)) {
-    if (FVC->getReturnVar())
-      getVarsFromConstraint(FVC->getReturnVar(), R);
+    if (FVC->getExternalReturn())
+      getVarsFromConstraint(FVC->getExternalReturn(), R);
     for (unsigned I = 0; I < FVC->numParams(); I++)
-      getVarsFromConstraint(FVC->getParamVar(I), R);
+      getVarsFromConstraint(FVC->getExternalParam(I), R);
```

Comment:
Are we always sure this function is used in a manner that external variables are the ones to get? I wonder if it makes sense to rename this function so it's obvious why the external ones are the ones we want.